models/deepseek/v4/prefill_swa_draft.py

```
@@ -0,0 +1,96 @@
+from __future__ import annotations
+
+import pypto.language as pl
+
+from prefill_attention_swa_draft import B, D, HC_MULT, S
+
+
+
+@pl.jit
+def prefill_swa(
+    x_hc: pl.Tensor[[B, S, HC_MULT, D], pl.BF16],
+    hc_attn_fn: pl.Tensor[[MIX_HC, HC_DIM], pl.FP32],
```

Ruff (0.15.13)

[error] Undefined name `MIX_HC`

(F821)